test/integration/error-is-clickable/test/index.test.js

```diff
@@ -23,14 +31,67 @@ describe('Clickable error link', () => {
 
     await hasRedbox(browser)
 
-    await browser.eval(`(function () {
-      document.querySelector("nextjs-portal")
-      .shadowRoot
-      .querySelector("#nextjs__container_errors_desc > a").click()
-      })()
-    `)
+    await evaluate(browser, () => {
+      document
+        .querySelector('body > nextjs-portal')
+        .shadowRoot.querySelector(
+          '#nextjs__container_errors_desc > a:nth-child(1)'
+        )
+        .click()
+    })
 
+    await waitFor(1000)
```

Comment:
We can't use arbitrary waits in tests, as this will lead to failures. TBH, this test probably doesn't need to check if the `browser.url()` changes, just that the element itself is an `<a>`.